### PR TITLE
Fixes for discord.com

### PR DESCRIPTION
**Pull request by Simsimpicpic**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1127,6 +1127,12 @@ CSS
 discord.com
 
 CSS
+path.wavePath-haxJK1 {
+   fill: #1D2021 !important;
+}
+.buttonBrand-33cvhX {
+   background-color: var(--brand);
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1134]: 4 spaces & `!important;`

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -1127,6 +1127,12 @@ CSS
 discord.com
 
 CSS
+path.wavePath-haxJK1 {
+   fill: #1D2021 !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1131]: 4 spaces

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -1127,6 +1127,12 @@ CSS
 discord.com
 
 CSS
+path.wavePath-haxJK1 {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1130]: This already has been fixed it can be removed.

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -1127,6 +1127,12 @@ CSS
...
+    fill: #1D2021 !important;
+}
+.buttonBrand-33cvhX {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1133]: Where can I find this?

Comment (Simsimpicpic) [reply src/config/dynamic-theme-fixes.config:1133]: This "waves"

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1133]: That's the wave that is been fixed by #3422

Comment (Simsimpicpic) [reply src/config/dynamic-theme-fixes.config:1133]: but... my pr is "26 days ago", so why you didn't merge ?

**Comment by Simsimpicpic** [comment]
This is not my problem

File: src/config/dynamic-theme-fixes.config (hunk 4 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1133]: Because I probably forgot.

**Pull request closed without merging**